lib/technical_metadata_service.rb

```diff
@@ -103,12 +103,8 @@ def dor_technical_metadata
   # @return [String] The datastream contents from the previous version of the digital object (fetched from preservation)
```

Comment:
Would you mind amending the YARD doc for `@return` here to reflect that (and why) `nil`s are returned?

Reply:
can do.

Reply:
done.